users/models.py

```diff
@@ -95,17 +95,17 @@ def __str__(self):
 
 
 class Apartment(models.Model):
-    owner = models.ForeignKey(AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='owner_apartments')
-    datePosted = models.DateTimeField(default=timezone.now, blank=True)
+    owner = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE)
+    date_posted = models.DateTimeField(default=timezone.now, blank=True)
     city = models.ForeignKey(City, on_delete=models.RESTRICT, related_name='city_apartments')
     address = models.TextField()
-    rentPricePerMonth = models.IntegerField()
```

Comment:
I think `Apartment` and` Seeker` shared fields names should be matched.
rentPricePerMonth - rent.
content - about.

We better select one and updated the needed model.
What do u think?
@tamirho

Reply:
Done